### PR TITLE
RELATED: RAIL-4296 Mention tip about visualization code generation

### PR DESCRIPTION
**Pull request by kandl**
- Now, you can generate the visualization code directly inside Analytical Designer - mention it in visualizations signpost article

JIRA: RAIL-4296

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1104/ : SUCCESS in 3m 32s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/740/ : SUCCESS in 6s

File: docs/02_start__choose_visualization_approach.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ The InsightView component allows you to embed insights created in [Analytical De
 It always reflects the current state of an insight: if any changes are made to the insight in Analytical Designer,
 the InsightView component in your application will pick them up and show the up-to-date version of the insight without any actions needed from your side.
 
+You can [generate the code](https://help.gooddata.com/pages/viewpage.action?pageId=86793524#EmbedInsights-Embedasaliveinsight) for particular InsightView directly in the Analytical Designer.
```
Comment (vendulalucakova) [inline docs/02_start__choose_visualization_approach.md:25]: .. for a particular InsightView directly in Analytical Designer.

File: docs/02_start__choose_visualization_approach.md [hunk 2]
```diff
@@ -48,6 +50,8 @@ GoodData.UI comes with several types of visualizations that you can use out of t
 They require more coding than the [InsightView component](#insightview-component) but at the same time offer more flexibility.
 For more information, see [Start with Visual Components](10_vis__start_with_visual_components.md).
 
+You can [generate the code](https://help.gooddata.com/pages/viewpage.action?pageId=86793524#EmbedInsights-Embedasapermanentinsight) for particular visual component directly in the Analytical Designer.
+
```
Comment (vendulalucakova) [inline docs/02_start__choose_visualization_approach.md:54]: .. for a particular InsightView directly in Analytical Designer.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1106/ : SUCCESS in 1m 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/742/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1108/ : SUCCESS in 3m 00s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/237/ : SUCCESS in 54s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/132/ : SUCCESS in 1m 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/327/ : SUCCESS in 21s

**Pull request merged**